core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -256,8 +256,8 @@ export class CustomizeInteractionModalComponent
   isSaveInteractionButtonEnabled(): boolean {
     return !!(
       this.hasCustomizationArgs &&
-      this.stateInteractionIdService.displayed &&
-      (this.getCustomizationArgsWarningsList().length === 0));
```

Comment:
@Shivkant-Chauhan I'm a bit concerned about removing this check. Wouldn't it allow, e.g., errors in other interactions (such as multiple-choice args) to get through?

If the error for EndExploration is non-blocking then perhaps we should treat that separately. But I think @EricZLou might have concerns about removing some of the frontend validation that (I assume) he's been trying to add recently to reduce errors getting to the backend.

/cc @EricZLou

Reply:
I think it won't affect other interactions because the function "isSaveInteractionButtonEnabled" is used within this file only, for the save interaction button of End exploration only.

Reply:
Hm, I don't think that's true. Could you please explain how you came to the conclusion in your last comment (i.e. "the function "isSaveInteractionButtonEnabled" is used within this file only, for the save interaction button of End exploration only")?

Reply:
Yes @seanlip, on removing this check, it is showing unusual behavior for other interactions also. I think it would be more appropriate to add a specific function for enabling the save interaction button for the end-exploration interaction only. I will be working on it and will give you the updates after that.

Reply:
Hi @seanlip PTAL. I have pushed some changes in the enabling condition of the save interaction button by adding a specific condition for the End Exploration and rest of the conditions are the same. Please re-review and tell me if any further changes are required. Thanks!